pkg/piperutils/FileUtils.go

```diff
@@ -47,3 +48,19 @@ func Copy(src, dst string) (int64, error) {
 	nBytes, err := io.Copy(destination, source)
 	return nBytes, err
 }
+
+func Download(src, dst string) (int64, error) {
+	source, err := http.Get(src)
```

Comment:
Maybe we should check for the response code here. From the docu of `net/http/Get``

>A non-2xx response doesn't cause an error

with this in mind we would continue e.g. after a `404` I suppose.

The name source is confusing. Of course that handle contains the source we would like to have, but first of all it is a response (having e.g. something like a body, headers, a response code). Maybe we should improve the naming so that gets clear that it is a response.

Reply:
Performing http downloads is some kind of an edge case for `FileUtils`. The code is better maintainable if it is well structured. I fear we end up in one gigantic `FileUtils`. Maybe we should introduce a `HttpUtils`.

Reply:
We do have our custom `http` package. Why not using this directly?

Reply:
as just discussed in person, let's add a `Downloader` interface to our `http` package